### PR TITLE
fix(SectionBreak, AvatarChooser): accessibility improvements, use roles & aria-live

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Add `role="separator"` to  GridFormSectionBreak component (part of GridForm organism)
Add `aria-live="polite"` to AvatarChooser container, and `role="alert"` to the FormError so the error msg is announced to a screen reader; 

I also tried  changing Choose Photo button of AvatarChooser to be an HTML `<button>` instead of a `<span role="button">` but that led to other errors I wasn't sure I wanted to look into now... 

To test, please see the sister PR in portal app
https://github.com/codecademy-engineering/portal-app/pull/1324
and the related preview environment:
https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1324

### PR Checklist

- [ ] Related to designs:
- [X] Related to JIRA ticket: [EGG-2142](https://codecademy.atlassian.net/browse/EGG-2142)
- [X] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.42-alpha.a483ab.0<br/>@codecademy/gamut-labs@28.6.5-alpha.a483ab.0<br/>@codecademy/gamut@43.1.1-alpha.a483ab.0<br/>@codecademy/monaco@0.5.31-alpha.a483ab.0<br/>@codecademy/styleguide@53.0.3-alpha.a483ab.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61e901136f1e42afcb5ac11a--gamut-preview.netlify.app>https://61e901136f1e42afcb5ac11a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61e901136f1e42afcb5ac11a>Deploy Logs</a></p>

**Review by JoshuaKGoldberg — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.42-alpha.a5c2c3.0<br/>@codecademy/gamut-labs@28.6.5-alpha.a5c2c3.0<br/>@codecademy/gamut@43.1.1-alpha.a5c2c3.0<br/>@codecademy/monaco@0.5.31-alpha.a5c2c3.0<br/>@codecademy/styleguide@53.0.3-alpha.a5c2c3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61ea4e0dc4f14c745240f181--gamut-preview.netlify.app>https://61ea4e0dc4f14c745240f181--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61ea4e0dc4f14c745240f181>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.42-alpha.552c79.0<br/>@codecademy/gamut-labs@28.6.5-alpha.552c79.0<br/>@codecademy/gamut@43.1.1-alpha.552c79.0<br/>@codecademy/monaco@0.5.31-alpha.552c79.0<br/>@codecademy/styleguide@53.0.3-alpha.552c79.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61ea5465ccd40795fb14445a--gamut-preview.netlify.app>https://61ea5465ccd40795fb14445a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61ea5465ccd40795fb14445a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.42-alpha.18b903.0<br/>@codecademy/gamut-labs@28.6.5-alpha.18b903.0<br/>@codecademy/gamut@43.1.1-alpha.18b903.0<br/>@codecademy/monaco@0.5.31-alpha.18b903.0<br/>@codecademy/styleguide@53.0.3-alpha.18b903.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61ea57dc71d12e89b8434e43--gamut-preview.netlify.app>https://61ea57dc71d12e89b8434e43--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61ea57dc71d12e89b8434e43>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.42-alpha.7e346d.0<br/>@codecademy/gamut-labs@28.6.5-alpha.7e346d.0<br/>@codecademy/gamut@43.1.1-alpha.7e346d.0<br/>@codecademy/monaco@0.5.31-alpha.7e346d.0<br/>@codecademy/styleguide@53.0.3-alpha.7e346d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61ea5a9d8a932b7ef1d437d4--gamut-preview.netlify.app>https://61ea5a9d8a932b7ef1d437d4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61ea5a9d8a932b7ef1d437d4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.42-alpha.ead9a0.0<br/>@codecademy/gamut-labs@28.6.5-alpha.ead9a0.0<br/>@codecademy/gamut@43.1.1-alpha.ead9a0.0<br/>@codecademy/monaco@0.5.31-alpha.ead9a0.0<br/>@codecademy/styleguide@53.0.3-alpha.ead9a0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61ea5e032f637795e34e6ca4--gamut-preview.netlify.app>https://61ea5e032f637795e34e6ca4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61ea5e032f637795e34e6ca4>Deploy Logs</a></p>

File: packages/gamut-labs/src/AvatarChooser/index.tsx [hunk 1]
```diff
@@ -117,7 +118,7 @@ export const AvatarChooser: React.FC<AvatarChooserProps> = ({
         })}
         aria-invalid={Boolean(error)}
       />
-      {error && <StyledFormError>{error}</StyledFormError>}
+      {error && <StyledFormError role="alert">{error}</StyledFormError>}
```
Comment (dreamwasp) [inline packages/gamut-labs/src/AvatarChooser/index.tsx:121]: curious if this should also be aria-live or if the alert role suffices here?

**Review by dreamwasp — APPROVED** [review]
love it 🙏

**Pull request merged**